var/spack/repos/builtin/packages/freesurfer/package.py

```diff
@@ -18,6 +18,9 @@ class Freesurfer(Package):
 
     maintainers("robgics")
 
+    version("7.4.1", sha256="eb6545d1ffdee17a90abd2e7dc444aa1091a6138e257f6f956a7ff214635b092")
+    version("7.4.0", sha256="6b65c2edf3b88973ced0324269a88966c541f221b799337c6570c38c2f884431")
+    version("7.3.2", sha256="58518d3ee5abd2e05109208aed2eef145c4e3b994164df8c4e0033c1343b9e56")
```

Comment:
@zzzoom would it be enough to add the latest version (v7.4.1) to fulfill your use case? Or would you also need the other two versions specifically?

Reply:
Sure, but why? According to the release notes there's no difference in handling those versions, and having all versions available in spack helps with reproducibility.